stubs/hdbcli/hdbcli/dbapi.pyi

```diff
@@ -38,7 +38,7 @@ class Connection:
     def rollback(self) -> None: ...
     def setautocommit(self, auto: bool = ...) -> None: ...
     def setclientinfo(self, key: str, value: str | None = ...) -> None: ...
-    def ontrace(self) -> None: ...
+    def ontrace(self, callback: Callable[[str], Any], options: str | None) -> None: ...
```

Comment:
Based on the docs, I think `options` may be omitted here, and if it's present it should be a string.

Something like `options: str = ...`

https://help.sap.com/docs/SAP_HANA_CLIENT/f1b440ded6144a54ada97ff95dac7adf/c7f3f3f2f1e94e0bae307360fc5af265.html